tools/WinObjC.Tools/vsimporter-templates/packageable.project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "WinObjC.Packaging": "*",
+    "WinObjC.Frameworks": "*",
```

Comment:
this is the one that will end up in the imported project's directory right?

---
In reply to: [100940627](https://github.com/Microsoft/WinObjC/pull/1958#discussion_r100940627) [](ancestors = 100940627)